src/main.ts

```diff
@@ -54,7 +54,8 @@ export const enum StandardTokenType {
 	Other = 0,
 	Comment = 1,
 	String = 2,
-	RegEx = 4
+	RegEx = 4,
+	MetaEmbedded = 8

```

Comment:
No encoded tokens will ever be marked as MetaEmbeded. It's just used internally to reset the scope. Let me know if you can think of a better way to express this

Reply:
Yes, this is problematic because this enum makes it in the public module API